model.py

```diff
@@ -1,150 +1,261 @@
+import sqlite3
+import json
+import random
+
 class Model:
```

Comment:
This class should be split into smaller ones, because it is quite extensive now. Currently, let's keep it like this for the purpose of the first merge.

Reply:
cool!

Reply:
we will keep this for now